in.go

```diff
@@ -63,7 +63,7 @@ func Get(request GetRequest, github Github, git Git, outputDir string) (*GetResp
 	}
 
 	// Create the metadata
-	var metadata Metadata
+	metadata := Metadata{}
```

Comment:
What is the reason for changing this? 😅

Reply:
To remove the warning `Goland` throws here :) for `Receiver 'metadata' may be nil in call`.